defaults/defaults.go

```diff
@@ -66,12 +66,15 @@ const (
 	ConnectivityCheckNamespace = "cilium-test"
 
 	ConnectivityCheckAlpineCurlImage = "quay.io/cilium/alpine-curl:v1.4.0@sha256:2550c747831ff575f2147149b088ea981c06f9b6bcd188756d1b82cc10997956"
+	ConnectivityPerformanceImage     = "quay.io/cilium/network-perf:bf58fb8bc57c4933dfa6e2a9581d3925c0a0571e@sha256:9bef508b2dcaeb3e288a496b8d3f065e8636a4937ba3aebcb1732afffaccea34"
 	ConnectivityCheckJSONMockImage   = "quay.io/cilium/json-mock:v1.3.0@sha256:2729064827fa9dbfface8d3df424feb6c792a0ba07117b844349635c93c06d2b"
 
 	ConfigMapName   = "cilium-config"
 	Version         = "v1.11.2"
 	HubbleUIVersion = "v0.8.5"
 
+	PerfTestDuration = 30
```

Comment:
This const currently does not seem to be used. Do we want to use it in the definition of the `--perf-duration` flag?